test/python/WMCore_t/Services_t/RequestManager_t/RequestManager_t.py

```diff
@@ -123,13 +126,12 @@ def testA_RequestManagerService(self):
         
         self.reqService.sendMessage(requestName,"error")
         self.reqService.putWorkQueue(requestName, "http://test_url")
-        self.reqService.reportRequestProgress(requestName)
         self.reqService.reportRequestProgress(requestName,
                         percent_complete = 100, percent_success = 90)
         
-        self.reqService.reportRequestStatus(requestName, "running-open")
+        self.reqService.updateRequestStatus(requestName, "running-open")
```

Comment:
Well, by logic it should be update request status. By code I think now they do
the same (reportRequsetStatus and updateRequestStatus). You better look at
RequestManager code and decide which method to use and keep. The difference
is that updateRequestStatus uses request/<REQUESTNAME> call while
reportRequestStatus uses request call to REST server.

On  0, ticoann <notifications@github.com> wrote:
> >          self.reqService.reportRequestProgress(requestName,
> >                          percent_complete = 100, percent_success = 90)
> >          
> > -        self.reqService.reportRequestStatus(requestName, "running-open")
> > +        self.reqService.updateRequestStatus(requestName, "running-open")
> 
> Does this need to be changed to updateRequestStatus?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6495/files#r49108226